src/core/Accordion.tsx

```diff
@@ -203,7 +203,10 @@ const Accordion = ({
             name={item.name}
             rowIcon={item.icon}
             open={fullyOpen ?? openIndexes.includes(currentIndex)}
-            onClick={() => handleSetIndex(currentIndex)}
+            onClick={() => {
+              handleSetIndex(currentIndex);
+              item.onClick?.();
```

Comment:
Something to consider here is what you would like to do with this `onClick` in your calculator in `voltaire` - perhaps it may be helpful to pass something back in this function instead of nothing, maybe the `currentIndex`?

This isn't a veiled request, I'm not saying that you _need_ to pass something back, you might not - but just think what may help you identify which button has been clicked in `voltaire` or anything else you may need. If you do need something, you'll need to adjust this and the type of `onClick` in `AccordionData`.